cterasdk/common/object.py

```diff
@@ -1,7 +1,16 @@
 # pylint: disable=too-many-instance-attributes
+import copy
 import json
 
 
 class Object:
+    _sdk_hidden = ['password']
```

Comment:
There are a few other fields in our schema that are sensitive:
awsSecretKey
sharedSecret
passPhraseSalt
encPassphrase
encryptedFolderKey
oldPassword
newPassword
secretkey
activationCode